packages/sdk/test/e2e/node/onBehalfOfUserCredential.spec.ts

```diff
@@ -31,7 +31,7 @@ let ssoToken: string;
mduaDVFPSIsImFtciI6WyJyc2EiLCJtZmEiXSwiYXBwX2Rpc3BsYXluYW1lIjoiR3JhcGggRXhwbG9yZXIiLCJhcHBpZCI6ImRlOGJjOGI1LWQ5ZjktNDhiMS1hOGFkLWI3NDhkYTcyNTA2NCIsImFwcGlkYWNyIjoiMCIsImNhcG9saWRzX2xhdGViaW5kIjpbIjU5NTZmZjVhLTZmZGItNDc3ZS05ZDRkLTlmN2QyNjJlNjk0YSJdLCJjb250cm9scyI6WyJhcHBfcmVzIl0sImNvbnRyb2xzX2F1ZHMiOlsiZGU4YmM4YjUtZDlmOS00OGIxLWE4YWQtYjc0OGRhNzI1MDY0IiwiMDAwMDAwMDMtMDAwMC0wMDAwLWMwMDAtMDAwMDAwMDAwMDAwIiwiMDAwMDAwMDMtMDAwMC0wZmYxLWNlMDAtMDAwMDAwMDAwMDAwIl0sImRldmljZWlkIjoiOTA0YjU3MGYtYzA2Zi00OTUxLTlkZmEtYTcwM2FkOTdmOTk0IiwiZmFtaWx5X25hbWUiOiJaaGFvIiwiZ2l2ZW5fbmFtZSI6IllpcWluZyIsImlkdHlwIjoidXNlciIsImlwYWRkciI6IjE4My4xOTQuMTY4LjE1OSIsIm5hbWUiOiJZaXFpbmcgWmhhbyIsIm9pZCI6ImY0N2FjZTIyLWU2OTctNDI0Yy05OWMzLWExOWRjNzVjMDA5YSIsIm9ucHJlbV9zaWQiOiJTLTEtNS0yMS0yMTI3NTIxMTg0LTE2MDQwMTI5MjAtMTg4NzkyNzUyNy01OTA4ODQxMiIsInBsYXRmIjoiMyIsInB1aWQiOiIxMDAzMjAwMjBGNEQyNDEyIiwicmgiOiIwLkFSb0F2NGo1Y3ZHR3IwR1JxeTE4MEJIYlJ3TUFBQUFBQUFBQXdBQUFBQUFBQUFBYUFEMC4iLCJzY3AiOiJDYWxlbmRhcnMuUmVhZFdyaXRlIENvbnRhY3RzLlJlYWRXcml0ZSBEZXZpY2VNYW5hZ2VtZW50QXBwcy5SZWFkV3JpdGUuQWxsIERldmljZU1hbmFnZW1lbnRDb25maWd1cmF0aW9uLlJlYWQuQWxsIERldmljZU1hbmFnZW1lbnRDb25maWd1cmF0aW9uLlJlYWRXcml0ZS5BbGwgRGV2aWNlTWFuYWdlbWVudE1hbmFnZWREZXZpY2VzLlByaXZpbGVnZWRPcGVyYXRpb25zLkFsbCBEZXZpY2VNYW5hZ2VtZW50TWFuYWdlZERldmljZXMuUmVhZC5BbGwgRGV2aWNlTWFuYWdlbWVudE1hbmFnZWREZXZpY2VzLlJlYWRXcml0ZS5BbGwgRGV2aWNlTWFuYWdlbWVudFJCQUMuUmVhZC5BbGwgRGV2aWNlTWFuYWdlbWVudFJCQUMuUmVhZFdyaXRlLkFsbCBEZXZpY2VNYW5hZ2VtZW50U2VydmljZUNvbmZpZy5SZWFkLkFsbCBEZXZpY2VNYW5hZ2VtZW50U2VydmljZUNvbmZpZy5SZWFkV3JpdGUuQWxsIERpcmVjdG9yeS5BY2Nlc3NBc1VzZXIuQWxsIERpcmVjdG9yeS5SZWFkV3JpdGUuQWxsIEZpbGVzLlJlYWRXcml0ZS5BbGwgR3JvdXAuUmVhZFdyaXRlLkFsbCBJZGVudGl0eVJpc2tFdmVudC5SZWFkLkFsbCBNYWlsLlJlYWRXcml0ZSBNYWlsYm94U2V0dGluZ3MuUmVhZFdyaXRlIE5vdGVzLlJlYWRXcml0ZS5BbGwgb3BlbmlkIFBlb3BsZS5SZWFkIFBvbGljeS5SZWFkLkFsbCBQcmVzZW5jZS5SZWFkIFByZXNlbmNlLlJlYWQuQWxsIHByb2ZpbGUgUmVwb3J0cy5SZWFkLkFsbCBTaXRlcy5SZWFkV3JpdGUuQWxsIFRhc2tzLlJlYWRXcml0ZSBVc2VyLlJlYWQgVXNlci5SZWFkQmFzaWMuQWxsIFVzZXIuUmVhZFdyaXRlIFVzZXIuUmVhZFdyaXRlLkFsbCBlbWFpbCIsInNpZ25pbl9zdGF0ZSI6WyJkdmNfbW5nZCIsImR2Y19jbXAiLCJrbXNpIl0sInN1YiI6InVlNExEeVR4VkxxWXQzOHVjMGlZejkza1RtV0xXV1RveUV0aGdBb05abUUiLCJ0ZW5hbnRfcmVnaW9uX3Njb3BlIjoiV1ciLCJ0aWQiOiI3MmY5ODhiZi04NmYxLTQxYWYtOTFhYi0yZDdjZDAxMWRiNDciLCJ1bmlxdWVfbmFtZSI6InlpcWluZ3poYW9AbWljcm9zb2Z0LmNvbSIsInVwbiI6InlpcWluZ3poYW9AbWljcm9zb2Z0LmNvbSIsInV0aSI6Im1jWkxieTd0dkU2Z0c0M2pDTndrQUEiLCJ2ZXIiOiIxLjAiLCJ3aWRzIjpbImI3OWZiZjRkLTNlZjktNDY4OS04MTQzLTc2YjE5NGU4NTUwOSJdLCJ4bXNfY2MiOlsiQ1AxIl0sInhtc19zc20iOiIxIiwieG1zX3N0Ijp7InN1YiI6IjlfbFZmcS1iYThRdUFlc1FVYVpNZnN3cV93YU85bnlaYVBvbnJ1UnhGbkEifSwieG1zX3RjZHQiOjEyODkyNDE1NDd9.L3XtmiFr7YpJrJR43TC-41ZK57_QMc8S3Yd6mq0eqEMSbc84NNI0BIt1gTJqNlGzmisTG6Q5pHbHz9UN7dZto2JxxQpwhsE02qIJ24SKTskQeYz63Wdv9EJ87chGMJl0k68s_uGgx62mIQTpUKKzXyvi5Wwa4vib-exuY9Xq3sgaGouElCqdc1rnPjRAXAKKJkQvsy4KU6ogmaozE4RFDv1m-EJeAAlQROG3MhHyzbvs4BxRQpGJRDiTnqV7DkelzPLSYBQkKsmXmWoLO3wXSeJX8hBwxokkicQsOxdmiupEw1vDdKpCLfaubMChbvuRbkIRsFIzkfrL3H8TNsq48g";
```

Comment:
it's better not use personal token for testing?

Reply:
Thanks, updated with "public" access token: https://learn.microsoft.com/en-us/graph/auth/auth-concepts#access-tokens